shtab/__init__.py

```diff
@@ -38,8 +38,8 @@
 SUPPORTED_SHELLS: List[str] = []
 _SUPPORTED_COMPLETERS = {}
 CHOICE_FUNCTIONS: Dict[str, Dict[str, str]] = {
-    "file": {"bash": "_shtab_compgen_files", "zsh": "_files", "tcsh": "f"},
-    "directory": {"bash": "_shtab_compgen_dirs", "zsh": "_files -/", "tcsh": "d"}}
+    "file": {"bash": "_shtab_compgen_files", "zsh": "_files", "tcsh": "f", "fish": None},
```

Comment:
I'm not sure what's the usage for these

Reply:
It's for `shtab.FILE` and `shtab.DIRECTORY` as per the `pathcomplete` example in https://docs.iterative.ai/shtab/use/#library-usage i.e.

```sh
pathcomplete <TAB> # should suggeat filenames
pathcomplete --dir <TAB> # should suggest directories
```

Reply:
I see; but there's no distinction between filenames and directories. `complete` in `fish` by default matches files, unless you explicitly tells it not to

Reply:
from https://fishshell.com/docs/current/completions.html#useful-functions-for-writing-completions it looks like these are required:

- `-f`
- `-F`
- `__fish_complete_directories`
- `__fish_complete_path`

Reply:
The `complete` command by default matches files, and `-f` flag tells it not to. I can confirm they matches files currently.